server/src/handlers/generator.rs

```diff
@@ -0,0 +1,51 @@
+use axum::{
+    response::{IntoResponse, Response},
+    http::header::HeaderValue,
+    body::Body,
+};
+use std::{convert::Infallible, fs::File, io::Read};
+use docx_rs::*;
+
+pub async fn handler() -> Result<impl IntoResponse, Infallible> {
+    let path = std::path::Path::new("./reglamento_interno.docx");
+    let mut file = match File::create(path) {
+        Ok(file) => file,
+        Err(_) => return Ok(Response::new(Body::empty())),
+    };
+    let header =
+        Paragraph::new().add_run(Run::new().add_text("Reglamento interno"));
+
+    let first_header =
+        Paragraph::new().add_run(Run::new().add_text("Sobre la cooperativa"));
+    
+    let chapter_paragraph=
+        Paragraph::new().add_run(Run::new().add_text("Algo sobre la cooperativa"));
+
+
+    if let Err(_) = Docx::new()
+        .add_paragraph(header)
+        .add_paragraph(first_header)
+        .add_paragraph(chapter_paragraph)
+        .build()
+        .pack(&mut file)
+    {
+        return Ok(Response::new(Body::empty()));
+    };
```

Comment:
Este codigo podria escribirse asi:
```suggestion
    let new_document_result = Docx::new()
        .add_paragraph(header)
        .add_paragraph(first_header)
        .add_paragraph(chapter_paragraph)
        .build()
        .pack(&mut file);

    if new_document_result.is_err() {
         return Err(GenerateDocumentError); 
    }
```
En lugar de devolver Ok deberiamos devolver un Err, creo que el endpoint no deberia devolver 200 si algo falla. [Acá](https://github.com/tokio-rs/axum/blob/main/examples/anyhow-error-response/src/main.rs) hay un ejemplo de como implemente un error


igual lo que estas haciendo es solo probar si el `Docx::new` falla y  despues no haces nada con el resultado si sale bien, eso lo hiciste a proposito?